### PR TITLE
test: CXTB-576 EHP - update email workflow my account modal 

### PR DESCRIPTION
**Pull request by cmoreno-nts**
-Added Test
-Added page object
image

### CODE REVIEW

File: tests/cases/employee_health_portal/case_employee_health_portal_account.yaml [hunk 1]
```diff
@@ -259,6 +259,123 @@ TestEmployeeInformationUpdateNumber:
       Strategy: xpath
       Id: $PAGE.employee_health_home_page.alert_updated_info$
 
+# CXTB-576 Update Email from My Account Portal
+TestEmployeeUpdateAccountEmail:
```
[the remark this answers is not in the document]
Comment (cmoreno-nts) [reply tests/cases/employee_health_portal/case_employee_health_portal_account.yaml:263]: Done

File: tests/cases/employee_health_portal/case_employee_health_portal_account.yaml [hunk 2]
```diff
@@ -259,6 +259,123 @@ TestEmployeeInformationUpdateNumber:
...
+      Strategy: xpath
+      Id: $PAGE.employee_health_my_account_modal.update_email_tab$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_my_account_modal.update_email_requirements_text$
+  # Add value for new email field
+    - Type: send_keys
+      Strategy: xpath
+      Id: $PAGE.employee_health_my_account_modal.update_email_new$
+      Value: $AND_CLI_employee_user2$
```
Comment (csanchez-nts) [inline tests/cases/employee_health_portal/case_employee_health_portal_account.yaml:341]: Change this by an alternative user

File: tests/cases/employee_health_portal/case_employee_health_portal_account.yaml [hunk 3]
```diff
@@ -259,6 +259,123 @@ TestEmployeeInformationUpdateNumber:
...
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_my_account_modal.my_info_tab$
+  # Click Update Email in left panel
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_my_account_modal.update_email_tab$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_my_account_modal.update_email_requirements_text$
```
Comment (csanchez-nts) [inline tests/cases/employee_health_portal/case_employee_health_portal_account.yaml:336]: Add actual text validation with an assert

Comment (cmoreno-nts) [reply tests/cases/employee_health_portal/case_employee_health_portal_account.yaml:351]: Done

File: tests/cases/employee_health_portal/case_employee_health_portal_account.yaml (hunk 2 above)
Comment (cmoreno-nts) [reply tests/cases/employee_health_portal/case_employee_health_portal_account.yaml]: Done

**Review by csanchez-nts — APPROVED** [review]

File: tests/cases/config.yaml [hunk 4]
```diff
@@ -157,5 +157,6 @@ Environments:
       alternative_correct_password: add_your_info #this password is used to update momentarily the current password
       alternative_incorrect_password: add_your_info #this password shouldn't comply with all requirements
 #Facility Leadership users
+      alternative_user: add_your_info
```
[the remark this answers is not in the document]
Comment (cmoreno-nts) [reply tests/cases/config.yaml]: Already fixed.

**Review by csanchez-nts — APPROVED** [review]

**Pull request merged**